Realm/RLMObjectBase.mm

```diff
@@ -50,7 +50,12 @@ static bool isManagedAccessorClass(Class cls) {
     return strncmp(className, accessorClassPrefix, sizeof(accessorClassPrefix) - 1) == 0;
 }
 
+static bool preventInitObjectSchema;
 static bool maybeInitObjectSchemaForUnmanaged(RLMObjectBase *obj) {
+    if (preventInitObjectSchema) {
```

Comment:
This change doesn't make sense to me. -[RLMDynamicObject sharedSchema] returns nil, so this function already does nothing for RLMDynamicObject. Maybe the Swift version is missing that override?

Reply:
removed and added the override for `sharedSchema` on `RealmSwiftObject`